packages/framework/src/Command/RedisCleanCacheCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use Redis;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RedisCleanCacheCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'shopsys:redis:clean-cache';
+
+    /**
+     * @var \Redis[]
+     */
+    private $redisClients;
+
+    /**
+     * @param \Redis[] $redisClients
+     */
+    public function __construct(array $redisClients)
+    {
+        $this->redisClients = $redisClients;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Cleans up redis cache');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        foreach ($this->redisClients as $redis) {
```

Comment:
I have an idea. What if you have some default client with redis prefix only. So you dont need foreach for all clients and you will be able to remove all cache.

Reply:
This is a good idea but it would require complete refactoring of the way we use Redis and I think that should not be part of this PR, so please create Issue or new PR.